sql/core/benchmarks/CollationNonASCIIBenchmark-results.txt

```diff
@@ -2,26 +2,53 @@ OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Linux 6.5.0-1017-azure
+UTF8_BINARY                                           548            550           2          0.1       13693.8      32.9X
+UNICODE_CI                                          14666          14677          16          0.0      366652.8       1.2X
 
 OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Linux 6.5.0-1017-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - compareFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ---------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                    18041          18047           8          0.0      451030.2       1.0X
-UNICODE                                              14023          14047          34          0.0      350573.9       1.3X
-UTF8_BINARY                                           1387           1397          14          0.0       34680.4      13.0X
-UNICODE_CI                                           14232          14242          14          0.0      355808.4       1.3X
+UTF8_BINARY_LCASE                                    18143          18174          44          0.0      453568.6       1.0X
+UNICODE                                              13869          13895          38          0.0      346713.1       1.3X
+UTF8_BINARY                                           1014           1016           3          0.0       25341.2      17.9X
+UNICODE_CI                                           14542          14546           5          0.0      363559.8       1.2X
 
 OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Linux 6.5.0-1017-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - hashFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                 10494          10499           6          0.0      262360.0       1.0X
-UNICODE                                           40410          40422          17          0.0     1010261.8       0.3X
-UTF8_BINARY                                        2035           2035           1          0.0       50877.8       5.2X
-UNICODE_CI                                        31470          31493          32          0.0      786752.4       0.3X
+UTF8_BINARY_LCASE                                 10002          10021          27          0.0      250062.0       1.0X
+UNICODE                                           39592          39623          45          0.0      989789.3       0.3X
+UTF8_BINARY                                        2029           2032           4          0.0       50718.3       4.9X
+UNICODE_CI                                        33172          33174           2          0.0      829309.4       0.3X
```

Comment:
I see that the benchmark runs on previous functions too, but we shouldn't make any unnecessary modifications in this PR (that are not related to the changes brought by this PR)

so I would advise keep the previous results as they were, and only append yours below

Reply:
I agree with you on that. But since those files are auto-generated it makes sense to store them like they were produced. However, this is a trivial change so I will revert those to previous values.